action.yml

```diff
@@ -306,23 +236,34 @@ runs:
         timeout_cmd: --timeout
         timeout: 600
 
-    - name: Release - on production - terraform apply [on master]
-      if: steps.infrastructure-changes.outputs.terraform == 'true' && github.ref == 'refs/heads/master'
-      working-directory: terraform
-      run: terraform apply -var 'environment=production' -var 'revision=${{ inputs.docker-image-tag }}' -auto-approve
-      shell: bash
+
 
     - name: Report - echo [on master]
       if: github.ref == 'refs/heads/master'
```

Comment:
we should add an extra condition when terraform changes are applied, we don't need to do a normal release when terraform apply is performed